### PR TITLE
Moved upgrade from FAQ to upgrade guide

### PR DESCRIPTION
**Pull request by chrisnegus**
The FAQ contained steps and information for upgrading Karpenter. This PR moves that information into the upgrade guide. I also did some editing of the existing upgrade guide.

### CODE REVIEW

File: website/content/en/preview/upgrade-guide/_index.md [hunk 1]
```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
...
 ## Security Patches
 
-While we are in major version 0 we will not release security patches to older versions, rather we provide the patches in
-the latest versions. When at major version 1 we will have an EOL (end of life) policy where we provide security patches
+While we are in major version 0 we will not release security patches to older versions.
+Rather we provide the patches in the latest versions.
+When at major version 1 we will have an EOL (end of life) policy where we provide security patches
 for a subset of older versions and deprecate the others.
 
-# Released Upgrade Notes
```
Comment (spring1843) [inline website/content/en/preview/upgrade-guide/_index.md:71]: Can you undo removing this and "There are no published upgrade notes yet."?

This is meant to be a section where we will link to upgrade notes about incompatibility when they become available.  This was stated earlier in the doc as "Add a permanent separate file named `migrating_to_vx.y.z.md` to our website (linked at the bottom of this page)"

File: website/content/en/preview/upgrade-guide/_index.md [hunk 2]
```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
 * Increment the minor version when in major version 0
 * Add a permanent separate file named `migrating_to_vx.y.z.md` to our website (linked at the bottom of this page)
   clearly explaining the breaking change and what needs to be done on the customer side to ensure a safe upgrade
```
Comment (spring1843) [inline website/content/en/preview/upgrade-guide/_index.md:40]: s/customer/user

File: website/content/en/preview/upgrade-guide/_index.md [hunk 3]
```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
...
 
 Besides the peer review process for all changes to the code base we also do the followings in order to find
 incompatibilities:
-* (To be implemented) Automated tests to see if installing, uninstalling, upgrading from an older version and downgrading
-to an older version to check the compatibility of the application
-* (To be implemented) Turn the commands in our
-documentation into scripts that we can automatically run to find incompatibilities between the application and
-documentation
+* (To be implemented) Automated tests for installing, uninstalling, upgrading from an older version, and downgrading to an older version to check the compatibility of the application
+* (To be implemented) Turn the commands in our documentation into scripts that we can automatically run to find incompatibilities between the application and documentation
```
Comment (spring1843) [inline website/content/en/preview/upgrade-guide/_index.md:48]: May be this should be "to check the compatibility of the documentation with the application" to be consistent with the bullet point

File: website/content/en/preview/upgrade-guide/_index.md [hunk 4]
```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
...
 
 ## Nightly Builds
 
-(To be implemented) Every night we will build and release everything that has been checked into the source code. This
-allows us to detect problems including breaking changes and potential drifts in our external dependencies sooner than we
-otherwise would. It also allows some of advanced users who have their own nightly builds to test the upcoming changes
-before they are released.
+(To be implemented) Every night we will build and release everything that has been checked into the source code.
+This allows us to detect problems including breaking changes and potential drifts in our external dependencies sooner than we otherwise would.
+It also allows some of advanced users who have their own nightly builds to test the upcoming changes before they are released.
```
Comment (spring1843) [inline website/content/en/preview/upgrade-guide/_index.md:54]: Should we change "allows" to "enables" to not repeat allows in to consecutive sentences?
Also should this be "some of our advanced users" or "some of Karpenter's advanced users?"?

File: website/content/en/preview/upgrade-guide/_index.md [hunk 5]
```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
...
 
 ## Release Candidates
 
-(To be implemented) We are considering having release candidates when we are in major version 1. By adopting this
-practice we allow our users who are early adopters to test out new releases before they are available to the wider
-community thereby providing us with early feedback which would result in more stable releases.
+(To be implemented) We are considering having release candidates when we are in major version 1.
+By adopting this practice we allow our users who are early adopters to test out new releases before they are available to the wider community thereby providing us with early feedback which would result in more stable releases.
```
Comment (spring1843) [inline website/content/en/preview/upgrade-guide/_index.md:59]: Should we change "which would result" to "resulting"?

**Review by spring1843 — CHANGES_REQUESTED** [review]

File: website/content/en/preview/upgrade-guide/_index.md (hunk 5 above)
Comment (chrisnegus) [reply website/content/en/preview/upgrade-guide/_index.md]: Done.

File: website/content/en/preview/upgrade-guide/_index.md (hunk 1 above)
Comment (chrisnegus) [reply website/content/en/preview/upgrade-guide/_index.md:71]: I added those lines back in.

File: website/content/en/preview/upgrade-guide/_index.md [hunk 6]
```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
...
-the latest versions. When at major version 1 we will have an EOL (end of life) policy where we provide security patches
+While we are in major version 0 we will not release security patches to older versions.
+Rather we provide the patches in the latest versions.
+When at major version 1 we will have an EOL (end of life) policy where we provide security patches
 for a subset of older versions and deprecate the others.
 
-# Released Upgrade Notes
+# Steps
+
+Upgrading your version of Karpenter requires that you:
```
[the remark this answers is not in the document]
Comment (chrisnegus) [reply website/content/en/preview/upgrade-guide/_index.md]: Okay. I removed the upgrade procedure.

**Review by chrisnegus — COMMENTED** [review]
I think I got all the comments in. Let me know if more changes are needed.

**Review by spring1843 — APPROVED** [review]
LGTM

**Pull request merged**